docs/developer-guide/release.md

```diff
@@ -15,7 +15,7 @@ Each new release of OVN-Kubernetes is defined with a "version" that represents t
   * `ovnkube-identity`: is the executable that is invoked to run ovn-kubernetes identity manager, which includes the admission webhook and the CertificateSigningRequest approver
```

Comment:
unrelated nit: https://ovn-kubernetes.io/developer-guide/release/#overview
if its not too much trouble could you please add an extra layer of indentation for the each binary bullet point so that it renders in as nested bullets?